scripts/deploy.sh

```diff
@@ -71,15 +81,21 @@ elif [[ ! -z "$dtag" && -z "$ptag" && -z "$rtag" ]]; then
         exit 1
     fi
 
-    # tag it locally
+    # Merge branch into master and push to origin
+    git checkout master
+    git pull origin
+    git merge --no-ff release_$rtag -m "Merge branch devel_$rtag"
+    git push origin master
+    git branch -d release_$rtag
+
```

Comment:
you might be right, @damianavila can you comment? One purpose of this PR is to make sure the BokehJS `version` is correct for *all* published artifacts, including dev builds and RCs